proto/eth/v1alpha1/slasher.pb.go

```diff
@@ -6,12 +6,11 @@ package eth
 import (
 	context "context"
 	fmt "fmt"
-	io "io"
-	math "math"
-
 	proto "github.com/gogo/protobuf/proto"
 	types "github.com/gogo/protobuf/types"
 	grpc "google.golang.org/grpc"
+	io "io"
+	math "math"
```

Comment:
So many of the imports got messed up, but these are compiled files right? Can you run gofmt and goimports?

Reply:
Just ran goimports and nothing changed on these files @0xKiwi , they should be fine because they are generated - in fact we shouldn't even check them into the repo at all, as they are generated by protogen with bazel at runtime

Reply:
These are fine as they are auto generated